.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -35,7 +31,7 @@ jobs:
         export sha=$(Build.SourceVersion)
         export GIT_BRANCH=$BUILD_SOURCEBRANCHNAME
         export FEEDSTOCK_NAME=$(basename ${BUILD_REPOSITORY_NAME})
-        export UPLOAD_ON_BRANCH="main"
+        export UPLOAD_ON_BRANCH="1.15.0rc0"
```

Comment:
You need to rerender the feedstock. You changed `upload_on_main` to `1.15.0rc1`, but the CI files are still using `1.15.0rc0`

Reply:
pushed ...